src/main/java/seedu/address/logic/commands/LoginCommand.java

```diff
@@ -42,6 +42,7 @@ public LoginCommand(Manager manager) {
     public CommandResult execute(Model model, CommandHistory history) throws CommandException {
         requireNonNull(model);
         if (model.loginUser(toLogin)) {
```

Comment:
maybe rename the method from `loginUser` to something like `isRegisteredUser` ... same thing for the 
method in `VersionedUsersList`.